action.yml

```diff
@@ -266,6 +266,7 @@ runs:
         INPUT_DIGGER_MODE: ${{ inputs.mode }}
         INPUT_DIGGER_COMMAND: ${{ inputs.command }}
         INPUT_DRIFT_DETECTION_SLACK_NOTIFICATION_URL: ${{ inputs.drift-detection-slack-notification-url }}
+        NO_BACKEND: ${{ inputs.no-backend }}
```

Comment:
you need to add `no-backend` in the list of parameters accepted by the action (above)

Reply:
fixed!